<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GDBackup</string>
  </property>
  <widget class="QWidget" name="miCentralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>611</width>
         <height>271</height>
        </rect>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>152</number>
       </attribute>
       <column>
        <property name="text">
         <string>Name folder</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Creation date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Assets</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Data</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>70</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>370</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Assets (songs, sfx)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="selectFolder">
       <property name="geometry">
        <rect>
         <x>510</x>
         <y>340</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Select folder</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>340</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Backup</string>
       </property>
      </widget>
      <widget class="QLabel" name="backupLocation">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>481</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"> border: 1px solid black;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 16pt;
</string>
       </property>
       <property name="text">
        <string>Backups</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>722</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAyuda_pibe">
    <property name="title">
     <string>Ayuda pibe</string>
    </property>
   </widget>
   <addaction name="menuAyuda_pibe"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
